server/helpers/tokenUtils.js

```diff
@@ -0,0 +1,31 @@
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+
+dotenv.config();
+
+const secret = process.env.JWT_SECRET;
+
+/**
+ *
+ * @param {integer} id - The id of the user
+ * @param {integer} lifeSpan - The the lifespan of the token
+ * @returns {void}
+ */
+export const createToken = (id, lifeSpan) => jwt
+  .sign({ id }, secret, { expiresIn: lifeSpan });
+
+/**
+ * Verifies the jwt and returns the payload
+ * @param {string} token - The jwt to be verified
+ * @returns {object} object containing user Id
+ */
+export const verifyToken = (token) => {
```

Comment:
This function seems to not be necessary. We can move the try-catch into the verifyJWT function in the middleware.

Reply:
Or better still combine this file and the verifyJWT middleware file into one.

Reply:
@mbilesanmi would that mean the `createToken` function too would be in the new file? I'm asking cause its a helper function, so we would have the `createToken` helper and the `verifyJWT` middleware in one file?

Reply:
@mbilesanmi can we discuss this during TTL interaction?